lib/galaxy/util/__init__.py

```diff
@@ -68,6 +67,8 @@
     Self,
 )
 
+from galaxy.util import requests
```

Comment:
Should be moved after the try/except imports (and can be rewritten as `from . import requests`).